src/styles/buildTextStyles.js

```diff
@@ -61,7 +61,7 @@ const buildTextStyles = css`
   }}
 
   /* Font Size */
-  font-size: ${({ size = 'medium' }) =>
+  fontSize:  ${({ size = 'medium' }) =>
```

Comment:
This is inside a string too. There's a few in this file.

Reply:
There is object below.

Reply:
Same as my other comment — I mean that this have been changed to camel case, but shouldn't it stay as `font-size` since it's in a string? If so, there's a few cases of this here.